index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint camelcase:0 */
```

Comment:
Please rename this file as cli.js, and update package.json to use cli.js for the cli